language-support/ts/daml-ledger/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Should we not just lint in CI and generate this on the fly as we do for daml2ts?

Reply:
No, please not. I would like to be able to have my IDE pick up the lint config as well. We also have three slightly different configs.